admin_manual/release_notes.rst

```diff
@@ -16,6 +17,102 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.8_release_notes_label:
+
+Changes in 10.0.8
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.0.8 that need your attention. You can also read `the full ownCloud Server changelog`_ for further details on what has changed.
+
+PHP 5.6 deprecation
+~~~~~~~~~~~~~~~~~~~
+PHP 5.6 active support has ended on 19 Jan 2017 and security support will be dropped by the end of 2018 (https://secure.php.net/supported-versions.php). Many libraries used by ownCloud (including the QA-Suite *PHPUnit*) will therefore not be maintained actively anymore which forces ownCloud to drop support in one of the next minor server versions as well. Please make sure to upgrade to PHP 7 soon. See the `system requirements in the ownCloud documentation <https://doc.owncloud.com/server/10.0/admin_manual/installation/system_requirements.html#officially-recommended-supported-options>`_.
```

Comment:
Given the end of active support for PHP 5.6, I'd add more emphasis to "_Please make sure to upgrade to PHP 7 soon._". I suggest something more like: "_Please make sure to upgrade to PHP 7 as soon as possible._"

Reply:
changed